src/Contracts/DocumentsQuery.php

```diff
@@ -117,6 +133,7 @@ public function toArray(): array
             'fields' => $this->getFields(),
             'filter' => $this->filter,
             'retrieveVectors' => (null !== $this->retrieveVectors ? ($this->retrieveVectors ? 'true' : 'false') : null),
+            'ids' => \is_array($this->ids) ? implode(',', $this->ids) : $this->ids,
```

Comment:
also what if empty array will be here?

Reply:
Updated the code and added a test. Empty arrays will be removed from the request.